src/driver/postgres/PostgresQueryRunner.ts

```diff
@@ -870,6 +870,8 @@ where constraint_type = 'PRIMARY KEY' and tc.table_catalog = '${this.dbName}'`;
                 c += " DEFAULT " + column.default + "";
             }
         }
+        if (column.isGenerated && column.type === "uuid")
+            c += " DEFAULT uuid_generate_v4()";
```

Comment:
is it only can be `uuid_generate_v4`? if there other functions like uuid_generate_v6 or something similar? If yes, then whats possibility that users may want to use that instead?